html/captive-portal/lib/captiveportal/PacketFence/Controller/Authenticate.pm

```diff
@@ -178,6 +179,24 @@ sub login : Local : Args(0) {
 
 }
 
+=head2 testLoginRetryLimit
+
+Limit the amount of time a user can retry a password
+
+=cut
+
+sub testLoginRetryLimit : Private {
+    my ( $self, $c ) = @_;
+    my $username = $c->request->param("username");
+    if($username) {
+        if ($self->reached_retry_limit($c, "login_retries", $c->profile->{'_login_attempt_retry_limit'})) {
+            $c->log->info("Max tries reached login code for $username");
+            $c->stash(txt_validation_error => i18n_format($GUEST::ERRORS{$GUEST::ERROR_MAX_RETRIES}));
```

Comment:
It doesn't show that message when you reach the limit.

It makes a silent failure.